com.unity.perception/Runtime/GroundTruth/Labelers/CameraLabeler.cs

```diff
@@ -60,12 +123,61 @@ internal void Init(PerceptionCamera newPerceptionCamera)
                 sensorHandle = newPerceptionCamera.SensorHandle;
                 Setup();
                 isInitialized = true;
+
+                if (supportsVisualization)
+                {
+                    InitPanel();
+                }
             }
             catch (Exception)
             {
                 this.enabled = false;
                 throw;
             }
         }
+
+        private GameObject GetCanvas()
+        {
+            var canvas = GameObject.Find("Canvas");
+            
+            if (canvas == null)
+            {
+               canvas = GameObject.Instantiate(Resources.Load<GameObject>("Canvas"));
```

Comment:
As we found with HDRP, interacting with the canvas requires an EventManager. Can we automatically create an EventSystem if it does not already exist?